mm2src/coins/lightning.rs

```diff
@@ -302,42 +459,75 @@ impl MarketCoinOps for LightningCoin {
         unimplemented!()
     }
 
+    // Todo: Implement this when implementing swaps for lightning as it's is used mainly for swaps
     fn tx_enum_from_bytes(&self, _bytes: &[u8]) -> Result<TransactionEnum, String> { unimplemented!() }
 
+    // Todo: When implementing swaps for lightning, maker_coin_start_block and taker_coin_start_block
+    // will not be used. So the return type for this function should be Option<u64> if we want to avoid
+    // unnecessary calls to get current_block. It can be kept as it is and won't cause a problem but we will do
```

Comment:
I only thought that returning `None` will be good because it represents that there is no such thing as blocks for LightningCoin.
I guess this function can just return `0` since the concept of blocks is not applicable to the lightning network, What do you think? `current_block` result will not be of any use to any functionalities relating to LightningCoin anyway.

Reply:
> I guess this function can just return 0

I think it's ok :sweat_smile:

Reply:
Done